src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -72,6 +73,14 @@ protected override void Initialize()
 
         protected Task LaunchScript(RequestContext<object> requestContext)
         {
+            // If this is a run without debugging session, disable all breakpoints.
+            if (this.noDebug)
+            {
+                Task disableBreakpointsTask = 
+                    this.editorSession.DebugService.DisableAllBreakpoints();
+                Task.WhenAll(disableBreakpointsTask);
```

Comment:
Task.WhenAll actually returns a Task that aggregates the await for a list of Tasks to complete.  You'd have to await it before it actually waits for completion.  The correct way to wait in this function would either be to change the function to async and await the `DisableAllBreakpoints` method or to just call `.Wait()` on the task that's returned from `DisableAllBreakpoints`.  The latter is a little more risky though because it can cause the